config/src/main/java/org/hyperledger/besu/config/GenesisConfigOptions.java

```diff
@@ -63,4 +67,10 @@
   OptionalInt getEvmStackSize();
 
   Map<String, Object> asMap();
+
+  OptionalLong getDefuseDifficultyBombBlockNumber();
```

Comment:
Also, keeping EIP-2124 in mind, could you also note in the comments which fork comes first?  And pull the ETC forks into a separate sequence of methods after getIstanbulBlockNumber in the "canonnical" order of events?

Reply:
I've updated the order of these in the code.